sympy/utilities/tests/test_lambdify.py

```diff
@@ -150,17 +159,46 @@ def test_numpy_transl():
         assert nump in numpy.__dict__
 
 
+def test_scipy_transl():
+    if not scipy:
+        skip("scipy is not installed.")
+
+    from sympy.utilities.lambdify import SCIPY_TRANSLATIONS
+    for sym, scip in SCIPY_TRANSLATIONS.items():
+        assert sym in sympy.__dict__
+        assert scip in scipy.__dict__
+
+
 def test_numpy_translation_abs():
     if not numpy:
-        skip("numpy not installed.")
+        skip("numpy is not installed.")
 
     f = lambdify(x, Abs(x), "numpy")
     assert f(-1) == 1
     assert f(1) == 1
 
+
+def test_scipy_translation_abs():
+    if not scipy:
+        skip("scipy is not installed.")
+
+    f = lambdify(x, Abs(x), "scipy")
+    assert f(-1) == 1
+    assert f(1) == 1
+
+
+# def test_numpy_translation_():
+#     if not numpy:
+#         skip("numpy is not installed.")
+
+#     f = lambdify(x, Abs(x), "numpy")
+#     assert f(-1) == 1
+#     assert f(1) == 1
```

Comment:
I found same code with different def name in [line : 172](https://github.com/sympy/sympy/pull/10319/files#diff-2220ba8f6c6427dab59fd14c133d009aR172) If it pass/fail first one ,then I don't think we need second def.